recipes/ecos/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "2.0.4" %}
+{% set sha256 = "1733e2e5f0fe12389acfaab9aff613d0405855c7aa802c1ca624652168255187" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#   $ openssl sha256 <file>
+# On Windows, you'll need the openssl pacakge, available on conda-forge
+
+package:
+  name: ecos
+  version: {{ version }}
+
+source:
+  fn: ecos-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/e/ecos/ecos-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.6
+    - scipy >=0.9
+
+  run:
+    - python
+    - numpy >=1.6
+    - scipy >=0.9
+
+test:
+  # Python imports
+  imports:
+    - ecos
+
+about:
+  home: http://github.com/embotech/ecos
+  license: GPLv3
```

Comment:
Please ask upstream to include the license file in `MANIFEST.in` to ensure it is included in `sdist`s and related packages.